### ISSUE TITLE
Adding a dev container from configuration files is stuck

### ISSUE THREAD

**Issue by jmeisele** [microsoft/vscode-dev-containers#1727]
VSCode Version: 1.76.0

Local OS Version: Ubuntu 22.04.2 LTS 64-bit

Logs: No logs or notification appear

Steps to Reproduce:

Open any new project without a devcontainer
Click on "Open a remote window" button
Click on : Add dev container configuration files
Click on ; From predefined container configuration definition
Select : NodeJS + TypeScript
Select node version is stuck and does not go to the next step

![image](https://user-images.githubusercontent.com/29043118/225002218-cb8af91e-282a-4675-8c4f-572ae080963f.png)

**Comment by jmeisele** [comment]
Same above holds true for macOS, and AWS Workspace Linux 2 OS. Tried downgrading devcontainer extension version and vscode version to previous versions and issue persists.

**Comment by ryanborMSFT** [comment]
Rolling back to `0.255.4` is a workaround at the moment: https://github.com/microsoft/vscode-dev-containers/issues/1724#issuecomment-1468190968

**Comment by rackerWard** [comment]
Mine is getting stuck at the container configuration template selection.  It gets stuck and stops responding if the "Azure Functions & C# - .NET 6 (In-Process)" template is selected.

I updated Docker today to the latest release.  I have not rebooted.  The first time I tried to create a container configuration VSCode gave me an error message that I don't have.  It does not prompt me the error message anymore.
![image](https://user-images.githubusercontent.com/77080415/225076236-0a03122b-097f-446c-b873-47c826d14876.png)

I am able to create a dev container selecting this template.

Another template is the "C# (.NET)" template. This template gets stuck when selecting the .NET version.
![image](https://user-images.githubusercontent.com/77080415/225076536-35299507-e044-4bf1-8e19-aef4eff4dfa0.png)

After attempting a few other templates it seems I am unable to execute the "add dev container configuration files"
